bin/include/functions.sh

```diff
@@ -40,12 +40,7 @@ javaMajorVersion() {
     if [ ${version} -eq 1 ]; then
         # Version seems starts from 1, we need second number.
         javaVersion "$1"
-        backIFS=$IFS
-
-        IFS=. ver=(${version##*-})
-        version=${ver[1]}
-
-        IFS=$backIFS
+        version=$(echo ${version} | awk -F[\"\.] '{print $2}')
```

Comment:
I would use the following notation:
`version=$(awk -F[\"\.] '{print $2}' <<< ${version})`